WordPress/Classes/ViewRelated/What's New/Views/GridCell.swift

```diff
@@ -0,0 +1,228 @@
+
+/**
+ A Cell which includes a header label and a horizontal stack view with multiple `Item`s
+
+     ┌───────────────┐
+     │┌─────────────┐│
+     ││headingLabel ││
+     │└─────────────┘│
+     │┌─────────────┐│
+     ││┌────┐ ┌────┐││
+     │││    │ │    │││
+     │││item│ │item│││
+     │││    │ │    │││
+     │││    │ │    │││
+     ││└────┘ └────┘││
+     ││gridStackView││
+     │└─────────────┘│
+     │ mainStackView │
+     └───────────────┘
+
+ See `makeItemStackView` for a description of `item`.
+*/
+class GridCell: UITableViewCell {
```

Comment:
Curious if there was an advantage to composing the UI via code vs a xib?

Reply:
I mostly just did this to match the `AnnouncementCell`. I don't have a strong preference here but I think we've been moving towards defining relatively small views in code.